### DISCUSSION TITLE
reconstitute MachineState Framework

### DISCUSSION THREAD

**Discussion by moonps1** [Digithought/DigiFramework#5] [Q&A]
hi,Allan
      i'm lucky to get this machineState framework.It's very strong and excellent.I use this dll to develop control software for industry machine.It's very nice.But recently our company with change developing language from c# to c++. This situation makes me very dispirited for a very long time......
     Do you have code this framework with cpp? If so,could you push that to github.
     Thank you very much.

**Comment by n8allan** [comment]
Hi, I'm really glad to hear that you've been successfully using
DigiFramework.  Unfortunately I don't have a C(++) version, but
architecturally I understand that the Qt Framework is pretty similar
(actors and HSMs).

Best of luck!

-Nate

**Reply by moonps1** [reply]
OK!  It's a new way for me.I'll explore it. TKS!
